proxy/Cargo.toml

```diff
@@ -17,5 +17,6 @@ serde_json = "1"
 tokio = { version = "1.7.1", features = ["full"] }
 tokio-postgres = "0.7.2"
 clap = "2.33.0"
+rustls = "0.19.1"
 
-zenith_utils = { path = "../zenith_utils" }
+zenith_utils = { path = "../zenith_utils" }
```

Comment:
missing a newline at the end of the file - not a big deal, but maybe worth fixing?